cmd/crates/soroban-spec-typescript/src/project_template/README.md

```diff
@@ -8,11 +8,11 @@ This library was automatically generated by Soroban CLI using a command similar
 soroban contract bindings ts \
   --rpc-url INSERT_RPC_URL_HERE \
   --network-passphrase "INSERT_NETWORK_PASSPHRASE_HERE" \
-  --id INSERT_CONTRACT_ID_HERE \
+  --contract-id INSERT_CONTRACT_ID_HERE \
   --name INSERT_CONTRACT_NAME_HERE
```

Comment:
two ux notes:

 - can we make name optional or derived somehow?
 - can we make the id a required positional arg? writing `--contract-id` is mildly annoying

Reply:
Ah, we just removed `--[contract]-name` in #849. Thanks for pointing this out; I'll remove this line.

Interesting question about making contract ID positional. That would make the full command look like:

```bash
soroban contract bindings typescript CB5T6MLZNWJBUBKEQAUVIG5JJWKYSYVVE2OVN25GMX3VX7CZ7OBAPAU4 \
  --network futurenet
  --output-dir path/to/contract-client
```

This introduces ambiguity, though, if you generate from a local `--wasm` rather than a deployed contract. I think maybe it's better to keep them as named arguments, for now.

Reply:
`--id` should be an alias, but for the most part we've opted for named parameters except in places like the config where it's clear what the main argument is.  I have been considering changing the invoke command to take --contract-id as a positional and then we could not need the slop `--` instead it would be seen as a subcommand.